libraries/angular/src/advanced-tests.js

```diff
@@ -78,6 +79,18 @@ describe("advanced support", function() {
       let data = wc.camelCaseObj;
       expect(data).to.eql({ label: "passed" });
     });
+
+    it("will pass object data to inherited properties", function() {
+      this.weight = 2;
+      let fixture = TestBed.createComponent(ComponentWithInheritance);
+      fixture.detectChanges();
+      let root = fixture.debugElement.nativeElement;
+      let wc = root.querySelector("#wc");
```

Comment:
Should we check bools, nums, and strings? Or do you want to do that in a basic test? Follow up PR?

Reply:
follow up PR?